docker/local-development.md

```diff
@@ -238,21 +239,104 @@ this:
 
 ```bash
 # The --rm flag will remove the temporary container created to run your command after the command exits.
-docker-compose run --rm <CONTAINER_NAME> <COMMAND>
+docker-compose run --rm ${CONTAINER_NAME} ${COMMAND}
 ```
 
 E.g., to run a specific test module:
 
 
 ```bash
-docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm app <YOUR_COMMAND>
+docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm app ${YOUR_COMMAND}
 ```
 
-– where `<YOUR_COMMAND>` is something like `pytest tests/test_admin.py -sxv --pdb`.
+– where `${YOUR_COMMAND}` is something like `pytest tests/test_admin.py -sxv --pdb`.
 
 Commands that write to STDOUT, e.g., `python manage.py dumpdata`, can be piped (`|`)
 or redirected (`>`) to files on your computer in the normal way.
 
+### Install a local installation of a Python package for development
+
+You can use containers to hack on Python packages, too! At DataMade, this comes
+up frequently when working on Councilmatic instances that require upstream
+changes to [`django-councilmatic`](https://github.com/datamade/django-councilmatic/),
+as well as writing or extending [custom comparators](https://github.com/dedupeio/?q=variable)
+for Dedupe.io. It's also super helpful for debugging third-party packages.
+
+To install a local copy of a Python package, first mount the directory it lives
+in as a volume your application container.
+
+**`docker-compose.yml`**
+
+```yaml
+app:
+    container_name: ${CONTAINER}
+    image: ${IMAGE}:latest
+    volumes:
+      - /absolute/path/to/your/local/package:/path/to/package/in/container
+    command: ${COMMAND}
+```
+
+Next, start your services:
+
+
+```bash
+docker-compose up -d
```

Comment:
In my testing of this process, in order to see the print statements I was adding, I didn't use detached mode. In general I find it easier for troubleshooting to just have `docker-compose up` running in a terminal window. Is there a reason you default to detached here?

Reply:
Personal preference. I'll strike `-d` and let people choose themselves.